src/utils/upload/fileUploadUtils.js

```diff
@@ -247,30 +253,74 @@ const fileUploadUtils = {
         return { valid: validFiles, invalid: invalidFiles };
       };
 
-      const { valid: DGEUnfilteredFiles, invalid } = getFilesMatching('DGE_unfiltered', files);
-      const { valid: DGEFilteredFiles } = getFilesMatching('DGE_filtered', invalid);
+      const dgeUnfilteredFiles = getFilesMatching('DGE_unfiltered/', files);
+      const dgeFilteredFiles = getFilesMatching('DGE_filtered/', dgeUnfilteredFiles.invalid);
+      const noMiddlePathFiles = getFilesMatching('', dgeFilteredFiles.invalid);
 
-      const filesToUpload = _.uniq([...Object.keys(DGEFilteredFiles), ...Object.keys(DGEUnfilteredFiles)])
+      // These are the ones that didn't match any of the 3 accepted shapes
+      const invalidFiles = noMiddlePathFiles.invalid.map((file) => ({
+        path: file.path,
+        rejectReason: 'Invalid file path. It should be in the form "sample/DGE_unfiltered/file", "sample/DGE_filtered/file" or "sample/file".',
+      }));
```

Comment:
how are we sure that all files that are invalid are invalid because of this particular reason?